config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-config.yaml

```diff
@@ -1,77 +1,3 @@
-periodics:
```

Comment:
Keep this line here, for the other periodic jobs that aren't deleted

Reply:
Thanks for noticing this.